cilium/cmd/config.go

```diff
@@ -75,6 +75,8 @@ func configDaemon(cmd *cobra.Command, opts []string) {
 
```

Comment:
I had added this change initially as part of this commit, but ended up not needing it. Figured it might be useful if we added more visibility into what the configuration of Cilium is as part of the API, but I'm open to removing it as part of this patch. What do you think?